Headers/DebugServer2/GDBRemote/Session.h

```diff
@@ -143,8 +143,8 @@ class Session : public SessionBase {
   void Handle_qOffsets(ProtocolInterpreter::Handler const &,
                        std::string const &);
   void Handle_qP(ProtocolInterpreter::Handler const &, std::string const &);
-  void Handle_qPlatform_IO_MkDir(ProtocolInterpreter::Handler const &,
-                                 std::string const &);
+  void Handle_qPlatform_mkdir(ProtocolInterpreter::Handler const &,
```

Comment:
It's not. I did a google search for the former, and the only results were some old lldb patches. A grep in the gdb and lldb sources for the former return nothing, and only the `qPlatform_mkdir` shows up in their docs. My assumption is that they started implementing and then decided it was a bad name.